### PR TITLE
bug 1546311. fix ocp image in downstream

### PR DESCRIPTION
**Pull request by jcantrill**
This PR updates Dockerfile to latest and adds ocp image

ref: https://bugzilla.redhat.com/show_bug.cgi?id=1546311

### CODE REVIEW

File: kibana/Dockerfile [hunk 1]
```diff
@@ -40,7 +39,22 @@ ADD probe/ /usr/share/kibana/probe/
...
+LABEL \
+        io.k8s.description="Kibana container for querying Elasticsearch for aggregated logs" \
+        com.redhat.component="logging-kibana-docker" \
+        vendor="Red Hat" \
+        name="openshift3/logging-kibana" \
+        License="GPLv2+" \
+        io.k8s.display-name="Kibana" \
+        version="v3.9.0" \
+        architecture="x86_64" \
+        release="0.46.0.0" \
```
Comment (ewolinetz) [inline kibana/Dockerfile]: this shouldn't be "1" as well?

**Comment by ewolinetz** [comment]
Can you add the link to the bz in the description?

File: kibana/Dockerfile (hunk 1 above)
Comment (jcantrill) [reply kibana/Dockerfile]: All of this will be regenerated by cd tooling

**Pull request merged**